tests/iscsi/iscsi_client.pm

```diff
@@ -0,0 +1,58 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "x11test";
+use strict;
+use testapi;
+use mm_network;
+use lockapi;
+
+sub run() {
+    my $self = shift;
+
+    x11_start_program("xterm -geometry 160x45+5+5");
+    become_root;
+    configure_default_gateway;
+    configure_static_ip('10.0.2.3/24');
+    type_string "yast2 iscsi-client\n";
+    assert_screen 'iscsi-client';
+    mutex_lock('iscsi_ready');    # wait for server setup
+    send_key "alt-i";             # go to initiator name field
+    sleep 2;
+    type_string "iqn.2016-02.de.openqa";
+    sleep 2;
+    assert_screen 'iscsi-initiator-service';
+    send_key "alt-v";             # go to discovered targets tab
+    sleep 2;
+    send_key "alt-d";             # press discovery button
+    sleep 2;
+    send_key "alt-i";             # go to IP address field
+    sleep 2;
+    type_string "10.0.2.1";
+    assert_screen 'iscsi-initiator-discovered-IP-adress';
+    send_key "alt-n";                                                            # next
+    assert_and_click 'iscsi-initiator-connect-button';                           # press connect button
+    assert_screen 'iscsi-initiator-connect-manual';
+    send_key "alt-n";                                                            # go to connected targets tab
+    assert_screen 'iscsi-initiator-discovered-targets';
+    send_key "alt-n";                                                            # next
+    assert_screen 'iscsi-initiator-connected-targets';
+    send_key "alt-o";                                                            # OK
+    sleep 2;
```

Comment:
We would like to use less sleep. Can you try to replace all "sleep"? I guess wait_still_screen works here.